app/allocation/views/view.njk

```diff
@@ -0,0 +1,97 @@
+{% extends "layouts/two-column.njk" %}
+
+{% set additionalContainerClass = "sticky-sidebar-container" %}
+
+{% block pageTitle %}
+  {{ t("allocations::view.page_title") }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: t("actions::back_to_dashboard"),
+    classes: "app-print--hide",
+    href: dashboardUrl
+  }) }}
+
+  <a href="javascript:window.print()" class="app-!-float-right app-print--hide govuk-!-margin-top-3 app-icon app-icon--print">
+    {{ t("actions::print_move_detail") }}
+  </a>
+
+  {{ super() }}
+{% endblock %}
+
+{% block contentHeader %}
+  <header class="govuk-!-margin-bottom-8 govuk-!-padding-left-0 govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-1">
+      {{ t("allocations::view.heading", {
+        person_or_people: "person" | pluralize(allocation.resultCount),
+        moves_count: allocation.moves_count,
+        from_location: allocation.from_location.title,
+        to_location: allocation.to_location.title,
+        date: allocation.date | formatDate
+      }) | safe }}
+    </h1>
+  </header>
+{% endblock %}
+
+{% block contentMain %}
+  <section>
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">{{ t("allocations::view.criteria_for_allocation") }}</h2>
+    <span class="govuk-caption-m govuk-!-margin-bottom-4">
+      {{ t("allocations::view.created_at") }}
+      {{ allocation.created_at | formatDateAsRelativeDay }}
+      {{ allocation.created_at | formatTime }}
+    </span>
+
+    {{ govukSummaryList(allocationDetails) }}
+
+    {% if allocation.complete_in_full and allocationPeople.emptySlots %}
+      {{ govukWarningText({
+        text: t("allocations::view.complete_in_full"),
+        iconFallbackText: "Warning"
+      }) }}
+    {% endif %}
+  </section>
+
+  <section class="govuk-!-margin-top-8">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">{{ t("allocations::view.people.heading") }}</h2>
+    <h3 class="govuk-heading-s">{{ t("allocations::view.people.slots_filled", {
+      slots_filled: allocationPeople.filledSlots.length,
+      total_slots: allocation.moves_count,
+      person_or_people: "person" | pluralize(allocation.moves_count)
```

Comment:
Again, could also look at i18next's [in-built pluralization](https://www.i18next.com/translation-function/plurals).

Reply:
Following up on this - maybe use the built-in pluralization support.